server/package.json

```diff
@@ -10,14 +10,16 @@
     "node": "^10.0"
   },
   "dependencies": {
+    "apollo-server": "^2.22.2",
+    "apollo-server-express": "^2.22.2",
     "body-parser": "^1.19.0",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "d3-dsv": "^1.1.1",
     "dataloader": "^1.4.0",
     "express": "^4.17.1",
     "express-graphql": "^0.6.12",
```

Comment:
We can drop the express-graphql and compression dependencies now, right? Any other packages we've stopped using here?

Reply:
Good point, forgot to remove express-graphql. 

We're still using compression to gzip responses, which are going to become even more redundant with batching.